dm/pkg/utils/common_test.go

```diff
@@ -257,3 +263,16 @@ func (s *testCommonSuite) TestNonRepeatStringsEqual(c *C) {
 	c.Assert(NonRepeatStringsEqual([]string{}, []string{"1"}), IsFalse)
 	c.Assert(NonRepeatStringsEqual([]string{"1", "2"}, []string{"2", "3"}), IsFalse)
 }
+
+func TestGoLogWrapper(t *testing.T) {
+	var wg sync.WaitGroup
+	wg.Add(1)
+	// to avoid data race since there's concurrent test case writing log.L()
+	l := log.Logger{Logger: zap.NewNop()}
+	go GoLogWrapper(l, func() {
+		defer wg.Done()
+		panic("should be captured")
+	})
+	wg.Wait()
+	require.True(t, true)
```

Comment:
I remember if a test panic, go will not treat it as pass. So this line is not needed